spec/event_store_client/store_adapter/api/client_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require_relative '../../event_store_helpers'
```

Comment:
can we include them in the `rspec_helper' ?

Reply:
Yes, we can.